docs/source/data/errors.mdx

```diff
@@ -40,16 +39,26 @@ Here's an example error response caused by misspelling the `__typename` field in
 
 </ExpansionPanel>
 
-To help with debugging, Apollo Server defines error subclasses that represent different types of errors that can occur while handling a GraphQL operation (such as `SyntaxError` and `ValidationError`). These subclasses each return a different [error code](#error-codes), which enables requesting clients to respond differently to different error types.
+To help with debugging, Apollo Server provides an `ApolloServerErrorCode` enum, which you can use to check if your error is one of the [different types recognized by Apollo Server](#error-codes).
+
+If Apollo Server recognizes your error type, you can write custom logic to handle that error, like so:
```

Comment:
Oops, that first comma should have been omitted. So this should read as:

```
If Apollo Server recognizes your error type you can write custom logic to handle that error, like so:
```

Reply:
I'm still not sure this makes sense. I think esp in this introductory place it would make more sense to say something more like "you can look at the `code` to act differently depending on what kind of error it was; the enum lists the various errors produced by Apollo Server itself, and you can also make up your own codes" or something?

AS itself is basically never "recognizing" error codes, just producing them.

Reply:
Gotcha, I see what you mean. 

How about something like:

> You can check an error's `code` to determine why an error occurred and also add logic to respond to different types of errors, like so: